doc/build_apps/get_started.rst

```diff
@@ -0,0 +1,504 @@
+(CCF) <https://ccf.dev/>`__ is an open-source framework for building a
+new category of secure, highly available, and performant applications
+that focus on multi-party compute and data.
+
+-  Read the `CCF
+   overview <https://microsoft.github.io/CCF/main/overview/index.html>`__
+   and get familiar with `CCF’s core
+   concepts <https://microsoft.github.io/CCF/main/overview/what_is_ccf.html>`__
+   and `Azure confidential
+   computing <https://learn.microsoft.com/en-us/azure/confidential-computing/>`__
+-  `Build new CCF
+   applications <https://microsoft.github.io/CCF/main/build_apps/index.html>`__
+   in TypeScript/JavaScript or C++
+-  CCF `Modules API
+   reference <https://microsoft.github.io/CCF/main/js/ccf-app/modules.html>`__
+-  CCF application get started repos
+
+   -  `CCF application
+      template <https://github.com/microsoft/ccf-app-template>`__
+   -  `CCF application
+      samples <https://github.com/microsoft/ccf-app-samples>`__
+
+Supported Programming Languages
+------------------------------
+
+Applications can be written in
+
+-  TypeScript
+-  JavaScript
+-  C++
+-  More language support upcoming in 2023
+
+Development environment
+-----------------------
+
+-  VS Code Dev Container |Open in VSCode|
+-  Github codespace: |Github codespace|
+-  Linux Machine (`Creating a Virtual Machine in Azure to run
+   CCF <https://github.com/microsoft/CCF/blob/main/getting_started/azure_vm/README.md>`__)
+
+
+(JavaScript/Typescript) Applications
+------------------------------------
+
+CCF apps can also be written in JavaScript or Typescript, To test a ccf
+application you need go through the following steps:
+
+-  Start a CCF Network with at least one node
+-  Initialize the CCF network with at least one (active member - user),
+   this can be done through `Network Governance
+   Proposals <https://microsoft.github.io/CCF/main/governance/proposals.html>`__.
+-  Create an application `deployment
+   proposal <https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html>`__
+-  Submit the app deployment proposal to the network and all members
+   accept it through voting. This is a part of `Network
+   Governance <https://microsoft.github.io/CCF/main/governance/proposals.html>`__.
+-  Open the CCF network for users
+-  Start to test your application endpoints
+
+Build Application
+~~~~~~~~~~~~~~~~~
+
+The application building prerequisites [`CCF <#ccf-install>`__, NodeJS
+and NPM] must be installed,all will be preinstalled if you are using
+devcontainer environment, otherwise you need to install them manually .
+
+In the checkout of
+`ccf-app-samples <https://github.com/microsoft/ccf-app-samples>`__
+repository:
+
+.. code:: bash
+
+   # carried out on the dev-container start up
+   cd banking-app && npm install
+
+   # build and generate the application bundle and deployment proposal
+   make build
+
+   # a dist folder is created with app bundle and deployment proposal.
```

Comment:
I think that a link to the `ccf-app-samples` repo should be enough here?

Reply:
Done